### PR TITLE
Update models.BUILD

### PR DESCRIPTION
**Pull request by cwolf22**
something here

### CODE REVIEW

**Comment by cwolfTester** [comment]
General comment

**Comment by cwolf22** [comment]
> General comment

Are you sure

File: models.BUILD [hunk 1]
```diff
@@ -17,3 +17,5 @@ filegroup(
     ),
 )
 
+
```
Comment (cwolfTester) [inline models.BUILD:20]: this looks good

File: models.BUILD [hunk 2]
```diff
@@ -17,3 +17,5 @@ filegroup(
...
+
```
Comment (cwolfTester) [inline models.BUILD:21]: this looks different

File: models.BUILD [hunk 3]
```diff
@@ -17,3 +17,5 @@ filegroup(
     ),
 )
```
Comment (cwolfTester) [inline models.BUILD]: comment elsewhere @cwolf22

Comment (cwolfTester) [inline models.BUILD]:
```suggestion
) // comment here
```

**Review by cwolfTester — CHANGES_REQUESTED** [review]
Overall pretty nice. but you need to make some updates

**Pull request merged**